app/src/main/java/org/cru/godtools/ui/dashboard/tools/ToolCategoriesAdapter.kt

```diff
@@ -10,6 +10,8 @@ import java.util.Locale
 import org.ccci.gto.android.common.androidx.recyclerview.adapter.SimpleDataBindingAdapter
 import org.cru.godtools.databinding.DashboardToolsCategoriesCategoryBinding
 
+const val CONSTANT_ALL_TOOLS = "all_tools"
```

Comment:
why define a constant, "All Tools" could easily be represented by `category = null` and doesn't introduce a special category constant you need to account for everywhere.